components/DataTable.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <data-view :title="title">
+    <template v-slot:button>
+      {{ `${chartData.datasets.length} 人` }}
+    </template>
+    <v-data-table
+      :headers="chartData.headers"
+      :items="chartData.datasets"
+      :items-per-page=-1
+      :hide-default-footer="true"
+      :height=200
+    ></v-data-table>
```

Comment:
```suggestion
    <v-data-table
      :headers="chartData.headers"
      :items="chartData.datasets"
      :items-per-page="-1"
      :hide-default-footer="true"
      :height="200"
    />
```

こまかなところですが！囲むものなければ閉じちゃうのと数字も""でやったほうが良いと思います(多分)
https://jp.vuejs.org/v2/guide/components-props.html#%E6%95%B0%E5%80%A4%E3%81%AE%E5%8F%97%E3%81%91%E6%B8%A1%E3%81%97